mozdownload/cli.py

```diff
@@ -119,6 +119,10 @@ def parse_arguments(argv):
                         help='Version of the application to be downloaded for release '
                              'and candidate builds (special values: %s)' % ', '.join(
                                 scraper.RELEASE_AND_CANDIDATE_LATEST_VERSIONS.keys()))
+    parser.add_argument('--print-url',
+                        action='store_true',
+                        help="Prints final url.")
```

Comment:
I would suggest to use 'Print final URL instead of downloading the file'. Please also add `dest` so we stay in sync with all the other options.